docs/integration.rst

```diff
@@ -1,12 +1,72 @@
 Integration
 ===========
 
+- :ref:`ReverseProxy`
 - :ref:`Azure`
 - :ref:`AWS`
 - :ref:`Databricks`
 - :ref:`GCP`
 
 
+.. _ReverseProxy:
+Reverse Proxy
+-------------
+
+Airflow can be set up behind a reverse proxy, with the ability to set its endpoint with great
+flexibility.
+
+For example, you can configure your reverse proxy to get:
+
+::
+
+    https://lab.mycompany.com/myorg/airflow/
+
+To do so, you need to set the following setting in your `airflow.cfg`::
+
+    web_server_url_prefix = /myorg/airflow
+
+Additionally if you use Celery Executor, you can get Flower in `/myorg/flower` with::
+
+    flower_url_prefix = /myorg/flower
+
+Your reverse proxy (ex: nginx) should be configured as follow:
```

Comment:
Add a note saying that the base_url is  (or whatever the correct setting is named) would be good here too.

Reply:
done